examples/simple_conduction/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level LICENSE file for
+# details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+blt_add_executable( NAME        simple_conduction_with_input_file
+                    SOURCES     with_input_file.cpp
+                    DEPENDS_ON  serac_physics serac_coefficients
+                    OUTPUT_NAME simple_conduction_with_input_file
```

Comment:
You shouldn't need an OUTPUT_NAME on either of these since BLT will default to NAME.  We do this on the executable because the driver and library CMake target name clash.